### PR TITLE
chore(ml): Exposed admin.machineLearning namespace

### PR DESCRIPTION
**Pull request by hiranya911**
Part of go/firebase-node-one-source proposal.

* Exposed `admin.machineLearning` namespace and the `App.machineLearning()` method.

### CODE REVIEW

File: src/machine-learning/index.ts [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+
+import { app } from '../firebase-namespace-api';
+
+/**
+ * Gets the {@link admin.machineLearning.MachineLearning `MachineLearning`} service for the
+ * default app or a given app.
+ *
+ * `admin.machineLearning()` can be called with no arguments to access the
+ * default app's {@link admin.machineLearning.MachineLearning
+  * `MachineLearning`} service or as `admin.machineLearning(app)` to access
```
Comment (lahirumaramba) [inline src/machine-learning/index.ts:25]: Nit: Indentation issue

File: src/machine-learning/index.ts [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+    listModels(options?: ListModelsOptions): Promise<ListModelsResult>;
+
+    /**
+     * Deletes a model from the current project.
+     *
+     * @param {string} modelId The ID of the model to delete.
+     */
+    deleteModel(modelId: string): Promise<void>;
+  }
+}
```
Comment (lahirumaramba) [inline src/machine-learning/index.ts:282]: Missing a new line

**Review by lahirumaramba — APPROVED** [review]
LGTM!
Left couple comments.

File: src/machine-learning/index.ts (hunk 1 above)
Comment (hiranya911) [reply src/machine-learning/index.ts]: Done

File: src/machine-learning/index.ts (hunk 2 above)
Comment (hiranya911) [reply src/machine-learning/index.ts]: Done

**Review by ifielker — APPROVED** [review]

**Pull request merged**